debian/changelog

```diff
@@ -1,10 +1,15 @@
-cloud-init (23.1.1-0ubuntu3) UNRELEASED; urgency=medium
+cloud-init (23.2-0ubuntu1) mantic; urgency=medium
 
   * d/control: Remove pep8 dependency. It is no longer used.
   * Upstream snapshot based on upstream/main at d6ac22e1.
     - Bugs fixed in this snapshot: (LP: #2011783, #2008888)
+  * Upstream snapshot based on 23.2.
+    List of changes from upstream can be found at
+    https://raw.githubusercontent.com/canonical/cloud-init/23.2/ChangeLog
+    - Bugs fixed in this snapshot: (LP: #1854594, #2016350, #1881925)
+      (LP: #2013967, #2011291)
```

Comment:
Only diff I have locally is consolidation of the unreleased "Upstream snapshot based on upstream/main...." bugs with the current upstream snapshot based on 23.2.

Resulting diff is here:
```suggestion
  * Upstream snapshot based on 23.2.
    List of changes from upstream can be found at
    https://raw.githubusercontent.com/canonical/cloud-init/23.2/ChangeLog
    - Bugs fixed in this snapshot: (LP: #2011783, #2008888, #1854594, #2016350)
      (LP: #1881925, #2013967, #2011291)
```

Reply:
I overlooked this. Amended.